src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -245,31 +304,52 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     this.setState({ isSearching: true }, this.debouncedFetchObjects);
   };
 
+  fetchWorkspace = () => {
```

Comment:
What about renaming to subscribe workspace changes?

Reply:
make sense, updated